src/components/common/Header/index.tsx

```diff
@@ -0,0 +1,342 @@
+                          ))}
+                        </ul>
+                      </div>
+                    </div>
+                  ))}
+                  <img
+                    className={cn({
+                      [styles.MobileHeader__logo]: true,
+                      [styles['MobileHeader__logo--bcsd']]: true,
+                    })}
+                    src="http://static.koreatech.in/assets/img/ic-bcsd_gray.png"
+                    alt="bcsd lab logo"
+                  />
+                  <img
+                    className={cn({
+                      [styles.MobileHeader__logo]: true,
+                      [styles['MobileHeader__logo--koin']]: true,
+                    })}
+                    src="http://static.koreatech.in/assets/img/rectangle_icon.png"
+                    alt="KOIN service logo"
+                  />
+                </nav>
+              ),
+              document.body,
+            )}
+          </>
+        ) : (
+          <>
+            <Link
+              className={styles.Header__logo}
+              to="/"
+              tabIndex={0}
+            >
+              <img src="https://static.koreatech.in/assets/img/logo_white.png" alt="KOIN service logo" />
+            </Link>
+            <div
+              className={cn({
+                [styles['Header__mega-menu']]: true,
+                [styles.MegaMenu]: true,
+              })}
+              onBlur={onBlurMegaMenu}
+              onMouseOut={onBlurMegaMenu}
+            >
+              <ul className={styles['MegaMenu__trigger-list']}>
+                {CATEGORY.map((category, index) => (
+                  <li
+                    key={category.title}
+                  >
+                    <button
+                      className={styles.MegaMenu__trigger}
+                      tabIndex={0}
+                      type="button"
+                      onClick={createOnChangeMenu(category.title)}
+                      onFocus={createOnChangeMenu(category.title)}
+                      onMouseOver={createOnChangeMenu(category.title)}
+                      aria-expanded={isMegaMenuExpanded}
+                      aria-controls={ID[`LABEL${index + 1}`]}
+                    >
+                      <span>
+                        {category.title}
+                      </span>
+                    </button>
+                  </li>
+                ))}
+              </ul>
+              <div
+                id={ID.PANEL}
+                className={styles.MegaMenu__panel}
+                onFocus={onFocusPanel}
+                onMouseOver={onFocusPanel}
+                aria-hidden={!isMegaMenuExpanded}
+                aria-labelledby={Array.from({ length: 2 }, (_, index) => ID[`LABEL${index + 1}`]).join(' ')}
+              >
+                <ul className={styles.MegaMenu__content}>
+                  {panelMenuList?.map((menu) => (
+                    <li className={styles.MegaMenu__menu}>
```

Comment:
```suggestion
                  {panelMenuList?.map((menu) => (
                    <li className={styles.MegaMenu__menu} key={menu.title}>
```
ker값이 없어서 에러날듯. 추가부탁해요~